### PR TITLE
Page Controls

### PR DESCRIPTION
**Pull request by MCurran16**
League Detail View

### CODE REVIEW

**Comment by hunterbrowning** [comment]
@MCurran16 Did you pull this off the footer branch? It looks like the footer code is in here already?

File: assets/js/vm/views/league-details/league-details.js.coffee [hunk 1]
```diff
@@ -22,5 +25,17 @@ do ->
...
 
+         ###-----------------------------------
+				Content Header Links
+         ----------------------------------###
+         @selectedDetailItem = ko.observable("details")
+
+
+      selectDetailItem: (detailItem) ->
+         @selectedDetailItem(detailItem)
+         console.log(detailItem)
```
Comment (hunterbrowning) [inline assets/js/vm/views/league-details/league-details.js.coffee]: So this is really the first time we have been able to address this, but when you create a PR we never want debugging console logs or window property creation remaining in the code. You will see a few strange circumstances in our code where we leave something for functionality that we are integrating a lot of things into (like auth callbacks) but its rare. 

Please remove.

**Comment by hunterbrowning** [comment]
The Google map is not displaying. I am seeing a 403 in the request for it in inspector. Whats up here?

**Comment by hunterbrowning** [comment]
The back error and the edge of this header are supposed to be inline on the sketch file. 
image

**Comment by hunterbrowning** [comment]
Hey also, if I constrain my window to be shorter than your content, your entire page is scrollable. Including the main header of the website which should not be the case. You can create a "issue" if you want and tackle that in a separate PR or do it here. Your call.

File: assets/js/vm/views/league-details/league-details.js.coffee (hunk 1 above)
Comment (MCurran16) [inline assets/js/vm/views/league-details/league-details.js.coffee]: oops. Fixed

**Comment by hunterbrowning** [comment]
LGTM

**Pull request merged**